src/composables/authSignUp.js

```diff
@@ -0,0 +1,27 @@
+import { createUserWithEmailAndPassword, updateProfile } from "firebase/auth";
+import { firebaseAuth } from "@/firebase/config";
+import { useState } from "react";
+
+function useAuthSignUp() {
+  const [addUser, setAddUser] = useState("");
+
+  const signup = ({ name, email, password }) => {
+    createUserWithEmailAndPassword(firebaseAuth, email, password)
+      .then((userCredential) => {
+        updateProfile(userCredential.user, { displayName: name });
+      })
+      .then((userCredential) => {
+        const user = userCredential.user;
+        setAddUser(user);
+        console.log("user signed up", user);
+      })
+      .catch((error) => {
+        const errorCode = error.code;
+        const errorMessage = error.message;
```

Comment:
Did you want to console log the error code and message here?

Reply:
@TaylorDJones11 You are just assigning variables here but not using them.

Reply:
I've updated this as well @jjleigh